### PR TITLE
Remove unnecessary cache invalidation for channel

### PR DESCRIPTION
**Pull request by agnivade**
```release-note
NONE
```

### CODE REVIEW

File: server/platform/services/sharedchannel/service.go [hunk 1]
```diff
@@ -261,7 +261,6 @@ func (scs *Service) notifyClientsForSharedChannelConverted(channel *model.Channe
 }
 
 func (scs *Service) notifyClientsForSharedChannelUpdate(channel *model.Channel) {
-	scs.platform.InvalidateCacheForChannel(channel)
```
Comment (agnivade) [inline server/platform/services/sharedchannel/service.go:264]: Hey @wiggin77 - I was looking at the cache invalidation code and this part looked a bit strange to me. From the caller, you are only updating the `SharedChannels` table. There is no change happening to the `Channels` table. Do we still need to invalidate the channel cache here? Or is there a bug where we have missed out on actually updating the channels table?

**Comment by agnivade** [comment]
Gentle ping @wiggin77

File: server/platform/services/sharedchannel/service.go (hunk 1 above)
Comment (wiggin77) [reply server/platform/services/sharedchannel/service.go:264]: I need to look deeper into whether a WS event is needed here at all.

**Review by wiggin77 — APPROVED** [review]

**Comment by agnivade** [comment]
Merging as e2e tests look good.

**Pull request merged**